gobblin-service/src/main/java/org/apache/gobblin/service/monitoring/DagActionStoreChangeMonitor.java

```diff
@@ -252,123 +234,14 @@ protected void processMessage(DecodeableKafkaRecord<String, DagActionStoreChange
     dagActionsSeenCache.put(changeIdentifier, changeIdentifier);
   }
 
-  protected void handleDagAction(String operation, DagActionStore.DagAction dagAction, String flowGroup,
-      String flowName, long flowExecutionId, DagActionStore.DagActionType dagActionType) {
-    // We only expect INSERT and DELETE operations done to this table. INSERTs correspond to any type of
-    // {@link DagActionStore.FlowActionType} flow requests that have to be processed. DELETEs require no action.
-    try {
-      switch (operation) {
-        case "INSERT":
-          handleDagAction(dagAction, false);
-          this.dagProcEngineMetrics.markDagActionsObserved(dagActionType);
-          break;
-        case "UPDATE":
-          // TODO: change this warning message and process updates if for launch or reevaluate type
-          log.warn("Received an UPDATE action to the DagActionStore when values in this store are never supposed to be "
-                  + "updated. Flow group: {} name {} executionId {} were updated to action {}", flowGroup, flowName,
-              flowExecutionId, dagActionType);
-          this.unexpectedErrors.mark();
-          break;
-        case "DELETE":
-          log.debug("Deleted dagAction from DagActionStore: {}", dagAction);
-          break;
-        default:
-          log.warn(
-              "Received unsupported change type of operation {}. Expected values to be in [INSERT, UPDATE, DELETE]",
-              operation);
-          this.unexpectedErrors.mark();
-          break;
-      }
-    } catch (Exception e) {
-      log.warn("Ran into unexpected error processing DagActionStore changes: ", e);
-      this.unexpectedErrors.mark();
-    }
-  }
```

Comment:
shouldn't we leave all of this impl as-is for the derived class to continue using?

that said, I do agree w/ making **the other form** of `handleDagAction` `abstract`